<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameHeader">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButtonGenerate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>mainToolButton</string>
     </property>
     <property name="text">
      <string>生成项目</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/images/res/gen.png</normaloff>:/images/res/gen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonConfig">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>mainToolButton</string>
     </property>
     <property name="text">
      <string>配置环境</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/images/res/config.png</normaloff>:/images/res/config.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonAddWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>mainToolButton</string>
     </property>
     <property name="text">
      <string>添加控件</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/images/res/add.png</normaloff>:/images/res/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonAboutUs">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>mainToolButton</string>
     </property>
     <property name="text">
      <string>关于我们</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/images/res/about.png</normaloff>:/images/res/about.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Resources.qrc">:/images/res/1484816114_Know.ico</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>11</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>titleLabel</string>
     </property>
     <property name="text">
      <string>个人知识库</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>780</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../Resources.qrc">:/images/res/lock.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>建议您在生成项目前先配置好项目</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>配置好项目资源可方便生成项目， 同时也可方便管理</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionTrayExit">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/images/res/exit.png</normaloff>:/images/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionTrayOpen">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/images/res/open.png</normaloff>:/images/res/open.png</iconset>
   </property>
   <property name="text">
    <string>打开主面板</string>
   </property>
   <property name="toolTip">
    <string>打开主面板</string>
   </property>
  </action>
  <action name="actionTrayLogin">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/images/res/login.png</normaloff>:/images/res/login.png</iconset>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录后可享全部功能</string>
   </property>
  </action>
  <action name="actionMenuProjectNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionMenuProjectConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionMenuProjectExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMenuHelpContactUs">
   <property name="text">
    <string>Contact us</string>
   </property>
  </action>
  <action name="actionMenuHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
